core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-workflow/dot-edit-content-sidebar-workflow.component.ts

```diff
@@ -15,13 +19,55 @@ import { DotMessagePipe } from '@dotcms/ui';
 @Component({
     selector: 'dot-edit-content-sidebar-workflow',
     standalone: true,
-    imports: [DotMessagePipe, SkeletonModule, SkeletonModule],
-    providers: [DotWorkflowService],
+    imports: [
+        DotMessagePipe,
+        SkeletonModule,
+        ButtonModule,
+        DialogModule,
+        DropdownModule,
+        FormsModule
+    ],
+
     templateUrl: './dot-edit-content-sidebar-workflow.component.html',
     styleUrl: './dot-edit-content-sidebar-workflow.component.scss',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DotEditContentSidebarWorkflowComponent {
+    /**
+     * Whether not exist content yet
+     *
+     * @type {boolean}
+     * @memberof DotEditContentSidebarWorkflowComponent
+     */
+    $isNewContent = input<boolean>(true, {
+        alias: 'isNew'
+    });
+    /**
+     * Whether the dialog should be shown.
+     *
+     * @type {boolean}
+     * @memberof DotEditContentSidebarWorkflowComponent
+     */
+    readonly $showDialog = model<boolean>(false, {
+        alias: 'showDialog'
+    });
+
+    /**
+     * Output event to select the workflow.
+     *
+     * @type {Output<string>}
+     * @memberof DotEditContentSidebarWorkflowComponent
+     */
+    readonly onSelectWorkflow = output<string>();
+
+    /**
+     * The selected workflow.
+     *
+     * @type {string}
+     * @memberof DotEditContentSidebarWorkflowComponent
+     */
+    $selectedWorkflow = model<SelectItem>({} as SelectItem);
```

Comment:
Avoid using as and an empty object here. I suggest using SelectItem | null instead in this case

Reply:
done